docs/manifest-releaser.md

```diff
@@ -517,7 +517,7 @@ Example:
     },
     {
       "type": "linked-versions",
-      "group-name": "my group",
+      "groupName": "my group",
```

Comment:
Drive-by doc update: as release-please actually accepts `groupName` and does not accept `group-name`